plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryClient.java

```diff
@@ -250,6 +254,11 @@ Job create(JobInfo jobInfo)
         return bigQuery.create(jobInfo);
     }
 
+    public void executeUpdate(ConnectorSession session, String query)
```

Comment:
`BigQueryClient` is initialized with`ConnectorSession` in `BigQueryClientFactory`. Can we move `ConnectorSession` to a field and avoid adding this argument to each method?

Reply:
This method is called from only two places. Can we migrate them to use another `executeUpdate(QueryJobConfiguration job)`?